_docs-sources/guides/production-framework/ingredients/automatic-updates/intro.md

```diff
@@ -0,0 +1,12 @@
+# Automatic Updates
```

Comment:
You've got several of these `intro` sections - Consider renaming these to `index.md` and then making these category pages. This way - the "Automatic Updates" line in the sidebar will be clickable. 

This will create a better lining experience as you will link to `/docs/guides/production-framework/ingredients/automatic-updates/` rather than having to also specify `...../intro`

Reply:
Seconded — this also creates much better search results, as the hit you get will be e.g. "Automatic Updates" instead of "Intro".

Reply:
Roger. Working on this now.

Reply:
Done: https://github.com/gruntwork-io/docs/pull/145/commits/caf08280001d15ae941013de33a5c6eb42a2b383